luminoso_api/json_stream.py

```diff
@@ -254,6 +260,13 @@ def _read_csv(reader, header, encode_fn):
         yield row_dict
 
 
+def convert_date(date_string, date_format):
+    if date_format != 'epoch':
+        return datetime.datetime.strptime(date_string, date_format).timestamp()
```

Comment:
I think it's probably better to complain visibly if the date seems nonsensical.  Although I don't know -- maybe we often get files where most of the dates are right and 1% of them are Barney the Purple Dinosaur.

In any case, if it is going to skip unparseable dates, it must not do so quietly.